diffkemp/diffkemp.py

```diff
@@ -468,7 +469,13 @@ def compare(args):
             old_fun_desc.mod.clean_module()
             new_fun_desc.mod.clean_module()
             LlvmModule.clean_all()
-
+    # Create yaml output
+    if output_dir is not None and os.path.isdir(output_dir):
+        old_dir_abs = os.path.join(os.path.abspath(args.snapshot_dir_old), "")
+        new_dir_abs = os.path.join(os.path.abspath(args.snapshot_dir_new), "")
+        YamlOutput(snapshot_dir_old=old_dir_abs, snapshot_dir_new=new_dir_abs,
+                   result=result).save(output_dir=output_dir,
+                                       file_name="diffkemp-out.yaml")
```

Comment:
Nit: this is quite a complex command, could we split into two commands with an intermediate variable?

Reply:
Sure, done.